zebra-network/src/protocol/external/codec.rs

```diff
@@ -24,6 +24,9 @@ use zebra_chain::{
 
 use crate::constants;
 
+#[cfg(test)]
+mod tests;
```

Comment:
Nitpick: We usually put the module declarations after the imports. (`use super` is an import.)

Reply:
Fixed in [`10c405c`](https://github.com/ZcashFoundation/zebra/pull/5835/commits/10c405ca251398f016e76909ecc970294bcb43d6).